AuctionServiceApp/src/main/java/com/EECS4413/AuctionServiceApp/client/CatalogueServiceClient.java

```diff
@@ -0,0 +1,16 @@
+package com.EECS4413.AuctionServiceApp.client;
+
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.*;
+
+import com.EECS4413.AuctionServiceApp.dto.ItemDTO;
+
+@FeignClient(name = "catalogue-service", url = "${catalogue.service.url}")
```

Comment:
Added Feign clients to interact with the other services (Catalogue Service). https://docs.spring.io/spring-cloud-openfeign/docs/current/reference/html/

Reply:
What's benefit of using this?

Reply:
@kingsley9

Reply:
Makes it easier to perform inter service communication cause there's less boilerplate code to implement than if we were using rest templates, which is the other option i've found for making api calls within services.

Reply:
@kingsley9 approved